src/btree/bt_vrfy.c

```diff
@@ -38,7 +39,8 @@ static int __verify_row_int_key_order(
   WT_SESSION_IMPL *, WT_PAGE *, WT_REF *, uint32_t, WT_VSTUFF *);
 static int __verify_row_leaf_key_order(WT_SESSION_IMPL *, WT_REF *, WT_VSTUFF *);
 static int __verify_tree(WT_SESSION_IMPL *, WT_REF *, WT_CELL_UNPACK *, WT_VSTUFF *);
-
```

Comment:
There's a bit of editing noise in this PR. Could you please take a quick scan through and remove unintentional changes?

Reply:
I've done so! Assuming you meant some of the other diffs, and not this line specifically?